addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js

```diff
@@ -1,4 +1,4 @@
-
+// @odoo-module keepraw
```

Comment:
As the banner just below this line states: this file is auto-generated and this change will be overwritten the next time we update the lib.
Any other way to prevent the transpilation?

Reply:
Oh, I didn't read the comment.
An other way to prevent the transpilation is to move the file out of src :/

Reply:
>  move the file out of src

Well, we have divergent needs :/ It needs to be in `src` because we want the translated terms to be extracted.
I'm not sure what we want to do.
Note that it's most probably ok if it's transpiled. It's just useless work. I don't know how fast/slow it is.